pkg/mgrconfig/config.go

```diff
@@ -51,6 +51,10 @@ type Config struct {
 	KernelBuildSrc string `json:"kernel_build_src,omitempty"`
 	// Is the kernel built separately from the modules? (Specific to Android builds)
 	AndroidSplitBuild bool `json:"android_split_build"`
+	// If true, ignore PCs returned by kcov that do not correspond to any instrumentation callbacks.
+	// This may help to work around errors in coverage report generation, but may as well cover bugs
+	// with corrupted coverage.
+	IgnoreMismatchingCoverage bool `json:"ignore_mismatching_coverage"`
```

Comment:
Perhaps it's better to make this the request parameter?
That was we don't pollute the config and also it's possible to use it retrospectively, e.g. if you did not know ahead of time that the instance will produce bad coverage. I guess it can also happen episodically (e.g. if the fuzzer somehow manages to inject garbage coverage). E.g. "/cover?force=1".

Reply:
Fair enough, let's do that.